### PR TITLE
small update

### PR DESCRIPTION
**Pull request by Rainer-Lang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Beside being blazing fast, minimizing the code you need to write, it is also rea
 ##A quick overview:
 - Click / Long-Click listeners
 - Selection / Multi-Selection
+  - RadioButton and CheckBox could be also used
```
Comment (mikepenz) [inline README.md]: ?? perhaps you add a sample :D

Comment (Rainer-Lang) [inline README.md]: Why not :)
I will add it later in this PR.

Comment (mikepenz) [inline README.md]: I will wait for the PR :D

**Comment by Rainer-Lang** [comment]
Merge?

**Pull request closed without merging**